src/react-components/layout/Row.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./Row.scss";
+import styleUtils from "../styles/style-utils.scss";
+import classNames from "classnames";
+
+export const Row = ({
+  children,
+  padding,
+  gap,
+  spaceBetween,
+  breakpointColumn,
+  topMargin,
+  flexClassName,
+  flexBasis,
+  noWrap = false,
+  childrenMarginR,
+  className
+}) => {
+  const paddingClass = padding === true ? styleUtils.lgPadding : styleUtils[`${padding}Padding`];
```

Comment:
This pattern where a `prop` can either be `true` or one of (`"lg"`, `"sm"`, `"md"`, etc) is strange to me. If the purpose is to provide a default value for `padding`, should we use [`defaultProps`](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values)?

Reply:
I also don't like this pattern at all -- I was following other components that do this. I'll change it to something that is better :P

Reply:
Since this pattern is already used in Row and other components, I'm going to merge this one as is, then make another PR where gap / padding for Row and Column are strings only.